client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
```

Comment:
You must be talking about https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/reducer.js#L92.
I think this shouldn't be an issue in this case since we use the site selectors to run our tests (and not the ones from `state/ui/selectors`). I'll have another check since I didn't consider the problem before 👍